data/changelog.txt

```diff
@@ -639,6 +681,10 @@ Effect enchantments can now apply to monsters
 Make JSON formatter replace nbsp with escape sequence
 Update houses to utilize palettes
 Added SMALL_HIDER flag to various monsters.
+Added MOVEMENT_EXERTION_MODIFIER as an enchantment
+Test case: NPCs prefer to use guns
+Pop out incompatible tools from workstations
+Rename 'fatigue' to 'sleepiness', mostly on the back end, to avoid confusing different ways of being knackered
```

Comment:
```suggestion
Test case: NPCs prefer to use guns
Rename 'fatigue' to 'sleepiness', mostly on the back end, to avoid confusing different ways of being knackered
```